--- vmd-1.9.4a38/configure	2019-10-17 09:11:20.000000000 -0700
+++ patch-files/configure	2019-11-06 12:26:16.096461975 -0800
@@ -476,7 +476,7 @@
 $arch_echo        = "echo";
 $arch_tar         = "tar";
 $arch_compress    = "compress";
-$arch_gnucompress = "/usr/local/bin/gzip";
+$arch_gnucompress = "gzip";
 $arch_latex       = "latex";
 $arch_compilerc   = $arch_echo;
 $arch_rcflags     = "No resource compiler required on this platform.";
@@ -706,8 +706,8 @@
 $fltk_defines     = "-DVMDGUI -DVMDFLTK";
 $fltk_dir         = "$vmd_library_dir/fltk";
 $fltk_include     = "-I$fltk_dir/include";
-$fltk_library     = "-L$fltk_dir/$config_arch";
-$fltk_libs        = "-lfltk -lX11";
+$fltk_library     = "-L$fltk_dir/lib";
+$fltk_libs        = "-lfltk -lX11 -lXfixes -lXrender -lXcursor -lXext -lXft -ltcl8.5 -lfontconfig";
 #@fltk_cc          = ('forms_ui.c');
 @fltk_cu          = ();
 @fltk_ccpp        = ( 'ColorFltkMenu.C',
@@ -1172,7 +1172,7 @@
 $libtachyon_defines     = "-DVMDLIBTACHYON";
 $libtachyon_dir         = "$vmd_library_dir/tachyon";
 $libtachyon_include     = "-I$libtachyon_dir/include";
-$libtachyon_library     = "-L$libtachyon_dir/lib_$config_arch";
+$libtachyon_library     = "-L$libtachyon_dir/lib";
 $libtachyon_libs        = "-ltachyon";
 @libtachyon_cc          = ();
 @libtachyon_cu          = ();
@@ -1395,7 +1395,7 @@
 $actc_defines     = "-DVMDACTC";
 $actc_dir         = "$vmd_library_dir/actc";
 $actc_include     = "-I$actc_dir/include";
-$actc_library     = "-L$actc_dir/lib_$config_arch";
+$actc_library     = "-L$actc_dir/lib";
 $actc_libs        = "-lactc";
 @actc_cc          = ();
 @actc_cu          = ();
@@ -1408,10 +1408,11 @@
 # OPTIONAL COMPONENT: NetCDF I/O Library (Used by cdfplugin)
 #######################
 $netcdf_defines     = "";
-$netcdf_dir         = "$vmd_library_dir/netcdf";
-$netcdf_include     = "-I$netcdf_dir/include";
-$netcdf_library     = "-L$netcdf_dir/lib_$config_arch";
-$netcdf_libs        = "-lnetcdf";
+$netcdf_dir         = "ROLLNETCDF";
+$hdf5_dir           = "ROLLHDF5";
+$netcdf_include     = "-I$hdf5_dir/include -I$netcdf_dir/include";
+$netcdf_library     = "-L$hdf5_dir/lib -L$netcdf_dir/lib";
+$netcdf_libs        = "$netcdf_dir/lib/libnetcdf.a $netcdf_dir/lib/libpnetcdf.a $hdf5_dir/lib/libhdf5.a $hdf5_dir/lib/libhdf5_hl.a $netcdf_dir/lib/libnetcdf.a $netcdf_dir/lib/libpnetcdf.a $hdf5_dir/lib/libhdf5.a $hdf5_dir/lib/libhdf5_hl.a -lcurl -lz";
 @netcdf_cc          = ();
 @netcdf_cu          = ();
 @netcdf_ccpp        = ();
@@ -1470,9 +1471,9 @@
 $stock_numpy_library_dir=$ENV{"NUMPY_LIBRARY_DIR"} || "$vmd_library_dir/python/lib_$config_arch/lib/python2.5/site-packages/numpy/core/include";
 
 $python_defines     = "-DVMDPYTHON";
-$python_include     = "-I$stock_python_include_dir -I$stock_numpy_include_dir -I$stock_numpy_library_dir";
-$python_library     = "-L$stock_python_library_dir";
-$python_libs        = "-lpython2.5 -lpthread";
+$python_include     = "-IPYTHON_INCLUDE_DIR -INUMPY_INCLUDE_DIR";
+$python_library     = "-LPYTHON_LIBRARY_DIR";
+$python_libs        = "-lpythonPYVERSION -lpthread";
 @python_h           = ('PythonTextInterp.h',
                        'VMDTkinterMenu.h',
 		       'py_commands.h',
@@ -2029,7 +2030,7 @@
   $arch_lex         = "flex";  # has problems with the vendor lex... 
 
   # also needed for plugins
-  $system_libs .= " -ldl";
+  $system_libs .= " -ldl -lXfixes -lXcursor";
 
   if ($config_cuda) {
 #    $arch_nvcc        = "/opt/nvidia/cudatoolkit/default/bin/nvcc";
