--- vmd-1.9.2/configure	2014-12-29 21:47:59.000000000 -0800
+++ patch-files/configure	2015-08-27 14:43:37.000000000 -0700
@@ -437,7 +437,7 @@
 $arch_echo        = "echo";
 $arch_tar         = "tar";
 $arch_compress    = "compress";
-$arch_gnucompress = "/usr/local/bin/gzip";
+$arch_gnucompress = "gzip";
 $arch_latex       = "latex";
 $arch_compilerc   = $arch_echo;
 $arch_rcflags     = "No resource compiler required on this platform.";
@@ -621,7 +621,7 @@
 $fltk_defines     = "-DVMDGUI -DVMDFLTK";
 $fltk_dir         = "$vmd_library_dir/fltk";
 $fltk_include     = "-I$fltk_dir/include";
-$fltk_library     = "-L$fltk_dir/$config_arch";
+$fltk_library     = "-L$fltk_dir/lib";
 $fltk_libs        = "-lfltk -lX11";
 #@fltk_cc          = ('forms_ui.c');
 @fltk_cu          = ();
@@ -953,7 +953,7 @@
 $libtachyon_defines     = "-DVMDLIBTACHYON";
 $libtachyon_dir         = "$vmd_library_dir/tachyon";
 $libtachyon_include     = "-I$libtachyon_dir/include";
-$libtachyon_library     = "-L$libtachyon_dir/lib_$config_arch";
+$libtachyon_library     = "-L$libtachyon_dir/lib";
 $libtachyon_libs        = "-ltachyon";
 @libtachyon_cc          = ();
 @libtachyon_cu          = ();
@@ -1142,7 +1142,7 @@
 $actc_defines     = "-DVMDACTC";
 $actc_dir         = "$vmd_library_dir/actc";
 $actc_include     = "-I$actc_dir/include";
-$actc_library     = "-L$actc_dir/lib_$config_arch";
+$actc_library     = "-L$actc_dir/lib";
 $actc_libs        = "-lactc";
 @actc_cc          = ();
 @actc_cu          = ();
@@ -1155,9 +1155,9 @@
 # OPTIONAL COMPONENT: NetCDF I/O Library (Used by cdfplugin)
 #######################
 $netcdf_defines     = "";
-$netcdf_dir         = "$vmd_library_dir/netcdf";
+$netcdf_dir         = "ROLLNETCDF";
 $netcdf_include     = "-I$netcdf_dir/include";
-$netcdf_library     = "-L$netcdf_dir/lib_$config_arch";
+$netcdf_library     = "-L$netcdf_dir/lib";
 $netcdf_libs        = "-lnetcdf";
 @netcdf_cc          = ();
 @netcdf_cu          = ();
@@ -1237,7 +1237,7 @@
 $python_defines     = "-DVMDPYTHON";
 $python_include     = "-I$stock_python_include_dir -I$stock_numpy_include_dir -I$stock_numpy_library_dir";
 $python_library     = "-L$stock_python_library_dir";
-$python_libs        = "-lpython2.5 -lpthread";
+$python_libs        = "-lpython2.7 -lpthread";
 @python_h           = ('PythonTextInterp.h',
                        'VMDTkinterMenu.h',
 		       'py_commands.h',
@@ -2254,7 +2254,7 @@
 
     # this is to make tcl happy
     # also needed for plugins
-    $system_libs .= " -ldl";
+    $system_libs .= " -ldl -lXfixes -lXcursor";
 
     if ($config_python) {
       # this is to make python happy
