--- plugins/Make-arch	2014-08-20 12:54:59.000000000 -0700
+++ patch-files/Make-arch	2015-03-20 15:27:40.988947838 -0700
@@ -401,15 +401,15 @@
 LINUXAMD64:
 	$(MAKE) dynlibs staticlibs bins \
 	"ARCH = LINUXAMD64" \
-	"COPTO = -fPIC -m64 -o " \
-	"LOPTO = -fPIC -m64 -o " \
-	"CC = gcc" \
-	"CXX = g++" \
+        "COPTO = -fPIC  -o " \
+        "LOPTO = -fPIC -o " \
+        "CC = ROLLCC" \
+        "CXX = ROLLCXX" \
 	"DEF = -D" \
-	"CCFLAGS = -m64 -O2 -fPIC -Wall" \
-	"CXXFLAGS = -m64 -O2 -fPIC -Wall" \
+	"CCFLAGS = -O2 -fPIC -Wall" \
+	"CXXFLAGS = -O2 -fPIC -Wall" \
 	"TCLLDFLAGS = -ltcl8.5 -ldl" \
-	"NETCDFLDFLAGS = -lnetcdf " \
+	"NETCDFLDFLAGS = -LROLLNETCDF -lnetcdf " \
 	"AR = ar" \
 	"NM = nm -p" \
 	"RANLIB = touch" \
